server/datastore/mysql/policies.go

```diff
@@ -495,8 +515,15 @@ func (ds *Datastore) ApplyPolicySpecs(ctx context.Context, authorID uint, specs
 			critical = VALUES(critical)
 		`
 		for _, spec := range specs {
```

Comment:
(Pending Mo's confirmation) You should also use `spec.FirstDuplicatePolicySpecName` here?